src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
@kwonoj Yeah, unfortunately it looks like we'll have to let this and #1288 stew for a while. :disappointed: